aunt-leahs-app/src/components/TableComponent.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import '../styles.css';
+import { Table, TableBody, TableCell, Paper, TableHead, TableRow } from '@material-ui/core';
+
+// REQUIRES: data prop has to be array of objects where each object key represents the name of the field in camelcase.
+//					 Refer to data in constants.jsx
+//					 each field in data should be either string or boolean. boolean fields are displayed '✓' or '✗'
+
+function changeHeaderTitle(title) {
```

Comment:
Change how? Can you explain what this function does? I can see that it capitalizes the first character of the input string. Does it do anything else? Why do we need this function? Is this function ever called?

Reply:
Here are examples:
firstName -> First Name
mailingList -> Mailing List

Yes, it is called at line 24. 
It does not do anything else.
We need it because we need to convert the name of each field of data into a header title.
I fixed the issued addressed in the other comments too!

Reply:
Thanks for explaining. I think formatHeaderTitle would be a little more clear.

Reply:
Changed it!